src/utils/api/ElectionManagement.ts

```diff
@@ -136,13 +136,18 @@ export async function deleteElection(electionId: string): Promise<Number> {
       headers: { Authorization: `JWT ${token}` },
     })
     .then((res) => {
-      if (res.status == 204) {
-        Notification['success']({
-          title: 'Success',
-          description: 'The election is deleted successfully',
-        });
-      }
+      Notification['success']({
+        title: 'Success',
+        description: 'The election has been successfully deleted',
+      });
       return res.status;
+    })
+    .catch((err) => {
+      Notification['error']({
+        title: 'Error',
+        description: 'Failed to delete the election',
+      });
+      return err.status;
```

Comment:
This is good for now.

Note to self for future:

May not be good to use frontend components in API/backend-interacting code.